src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -115,30 +119,27 @@ local function save(_sPath)
     return ok, err, fileerr
 end
 
-local tKeywords = {
-    ["and"] = true,
-    ["break"] = true,
-    ["do"] = true,
-    ["else"] = true,
-    ["elseif"] = true,
-    ["end"] = true,
-    ["false"] = true,
-    ["for"] = true,
+local tFunctions = {
     ["function"] = true,
-    ["if"] = true,
-    ["in"] = true,
-    ["local"] = true,
-    ["nil"] = true,
-    ["not"] = true,
-    ["or"] = true,
-    ["repeat"] = true,
-    ["return"] = true,
-    ["then"] = true,
-    ["true"] = true,
-    ["until"] = true,
-    ["while"] = true,
+    ["_G"] = true
 }
 
+local function addHighlights( tApi, sName )
```

Comment:
I'd rather avoid highlighting global functions for now. I understand the appeal of highlighting builtins, but would rather not for now.

Reply:
Why not?

Reply:
Any such approach is always going to be a "best guess". This won't correctly handle cases like `local read` (which do show up), and at that point it'll just end up being confusing.